sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -17,6 +17,8 @@ namespace Azure.AI.FormRecognizer
     public class FormRecognizerClient
     {
         internal readonly ServiceClient ServiceClient;
+        internal readonly ClientDiagnostics Diagnostics;
```

Comment:
Making it `internal` so the `Operation` instance can call `Diagnostics.CreateRequestFailedException`.